graylog2-web-interface/src/components/field-analyzers/FieldQuickValues.jsx

```diff
@@ -157,11 +166,11 @@ const FieldQuickValues = React.createClass({
     if (this.state.field !== undefined) {
       this.setState({ loadingData: true });
       if (this.state.showHistogram) {
-        FieldQuickValuesActions.getHistogram(this.state.field, this.state.fieldQueryObjects, this.state.options).then(() => {
+        FieldQuickValuesActions.getHistogram(this.state.field, this.state.fieldQueryObjects, this.state.options).finally(() => {
```

Comment:
Unfortunately this is not enough to remove the loading spinner if there was an error getting the histogram, as there will be no data at that point. As this is disabled now for ES 2.x users, I guess it's not a blocker, but still annoying if someone runs into it.

Reply:
I'm just wondering, why don't we use the `loading` state from the store in here?